packages/adapter-static/test/apps/prerendered/test/test.js

```diff
@@ -0,0 +1,27 @@
+import * as fs from 'node:fs';
+import { expect, test } from '@playwright/test';
+
+const cwd = process.cwd();
+
+test('generates HTML files', () => {
+	expect(fs.existsSync(`${cwd}/build/index.html`)).toBeTruthy();
+});
```

Comment:
feels kinda weird to use playwright for tests like these, no? should we split the tests up, or do you think that adds too much complexity?

Reply:
yeah, it did feel slightly weird, but there are so few of them. I think it's simpler with just a single test runner. we can always revisit if we add more non-browser tests

Reply:
fair enough — that weirdness aside, this is much nicer